### PR TITLE
Add config packageName support to RobolectricTestRunner

### PR DESCRIPTION
**Pull request by sparkym3**

### CODE REVIEW

**Comment by sparkym3** [comment]
It looks like the CI failed because it couldn't automatically accept the install of the new version of the android tools.

File: robolectric/src/main/java/org/robolectric/RobolectricTestRunner.java [hunk 1]
```diff
@@ -330,7 +339,7 @@ protected AndroidManifest getAppManifest(Config config) {
       AndroidManifest appManifest;
       appManifest = appManifestsByFile.get(identifier);
```
Comment (lexs) [inline robolectric/src/main/java/org/robolectric/RobolectricTestRunner.java]: Do you need to add the package name to the identifier (which is used for caching)?

Comment (sparkym3) [inline robolectric/src/main/java/org/robolectric/RobolectricTestRunner.java]: Yeah, I think that would make sense. I will get the pull updated.

Comment (sparkym3) [inline robolectric/src/main/java/org/robolectric/RobolectricTestRunner.java]: The pull has been updated to include the package name as part of the cache identifier.

**Comment by erd** [comment]
I made some changes that should hopefully fix the Travis build. Once it's back to green, would you mind rebasing?

**Comment by sparkym3** [comment]
I have rebased and it appears that the java7 tests pass but the java8 tests fail due to reasons that appear unrelated to the changes I have made. If you see a reason why my change would have affected the java8 tests then let me know.

**Comment by erd** [comment]
I re-ran the tests and everything looks fine. Thanks!

**Pull request merged**